<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HtmlCreator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Images/html-icon.png</normaloff>../../Images/html-icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>8000000.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">&lt;color name=&quot;black&quot;&gt;#00000000&lt;/color&gt;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark properties you want in your website</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="titleBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="headingBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Headings</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="paragraphBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Paragraphs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="imageBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="linkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Links</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="listBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lists</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="formBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Forms</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
